redisinsight/ui/src/pages/workbench/components/enablament-area/EnablementArea/components/InternalPage/InternalPage.tsx

```diff
@@ -120,6 +120,7 @@ const InternalPage = (props: Props) => {
         </div>
       </EuiFlyoutHeader>
       <div
+        data-testid="enablement-area__page"
```

Comment:
please, move `data-testid` at the end of attributes

Reply:
moved